examples/connect/webhook_ngrok.py

```diff
@@ -0,0 +1,95 @@
+# ---------------------------------------------------------------------------
+# Mbed Cloud Python SDK
+# (C) COPYRIGHT 2017 Arm Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# --------------------------------------------------------------------------
+"""Example showing extended usage of the webhook functionality.
+
+Prerequisites
+
+Two third party tools are used in this example:
+    hug - a simple python3 webserver
+    ngrok - a tool to tunnel public http requests to localhost
+
+Install python libraries:
+    pip install ngrok hug
+Install ngrok from https://ngrok.com/
+Follow the ngrok instructions to configure a tunnel
+Set the resulting tunnel url in the variable below
+Run the example with:
+   hug -f examples/webhook_ngrok.py
+Finally, visit 127.0.0.1:8000 in your browser to initiate the sequence
+"""
+from mbed_cloud.connect import ConnectAPI
+
+import hug
+
+import threading
+import traceback
+
+api = ConnectAPI()
+ngrok_url = 'https://8b5e7ff1.ngrok.io'
```

Comment:
Where does this URL come from?

Reply:
Magically appears from ngrok. It could be sys.argv'd. I'll check/improve naming and comments with the diagrams.

Reply:
Cool, thanks.